Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity

```diff
@@ -1299,6 +1299,16 @@ PrefabInstance:
   m_Modification:
     m_TransformParent: {fileID: 1456262916}
     m_Modifications:
+    - target: {fileID: 6475316257701582328, guid: 1093263a89abe47499cccf7dcb08effb,
```

Comment:
Are there prefabs we should update these on? Or do we want to leave those as-is and just update it in the scene?

Reply:
The prefabs are actually set up correctly, the ones in scene had their underlying models changed to be arranged vertically instead of horizontally, however, when they were set up before we added the touchable area, meaning those never got realigned.